### DISCUSSION TITLE
Example setup of an MQTT server in docker-compose

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#3265] [Ask A Question] [answered]
Would someone be kind enough to share their docker-compose.yml that includes setup of an MQTT server? I am not using Home Assistant at the moment, and I am relatively new to docker, MQTT and frigate. The docs assume use of a Home Assistant MQTT server.

**Comment by NickM-27** [comment, marked as answer]
This guide shows how to install mqtt: https://techoverflow.net/2021/11/25/how-to-setup-standalone-mosquitto-mqtt-broker-using-docker-compose/

Then you'll just need to follow the frigate docs to get the frigate service running as well: https://docs.frigate.video/installation/

**Reply by thinkloop** [reply]
Ty

**Reply by rozgonyiadam** [reply]
Please be careful with the referenced documentation. The line included in it is incorrect because it points to itself! Incorrect line: `password_file /mosquitto/config/mosquitto.conf`
